Thunder/utils/render_template.py

```
@@ -13,18 +13,36 @@
 from Thunder.utils.logger import logger
 from Thunder.vars import Var
 
-template_env = Environment(
+template_env = Environment(
```

Ruff (0.15.6)

[error] By default, jinja2 sets `autoescape` to `False`. Consider using `autoescape=True` or the `select_autoescape` function to mitigate XSS vulnerabilities.

(S701)